src/tests/functional/shared_test_classes/include/shared_test_classes/base/ov_subgraph.hpp

```diff
@@ -61,6 +61,16 @@ class SubgraphBaseTest : public CommonTestUtils::TestsCommon {
     virtual std::vector<ov::Tensor> get_plugin_outputs();
 };
 
+inline std::vector<InputShape> static_shapes_to_test_representation(const std::vector<ov::PartialShape>& shapes) {
```

Comment:
If i am correct, Shape is inherited from `Partial shape`

Reply:
That'll be too easy for us)
Shape is [derived](https://github.com/openvinotoolkit/openvino/blob/8f05bab59445a5389451886d7113cb99d66f0aa4/src/core/include/openvino/core/shape.hpp#L21) from vector<size_t> and PartialShape is [not derived](https://github.com/openvinotoolkit/openvino/blob/8f05bab59445a5389451886d7113cb99d66f0aa4/src/core/include/openvino/core/partial_shape.hpp#L31) from anything.